app/Exceptions/Csrf.php

```diff
@@ -16,14 +16,15 @@ public function __construct($message = '', $code = 0, \Exception $previous = nul
 		parent::__construct($message, $code, $previous);
 		\App\Session::init();
 		$userName = \App\Session::get('full_user_name');
+		$userName = empty($userName) ? '-' : \App\TextParser::textTruncate($userName, 100);
 		\App\Db::getInstance('log')->createCommand()
 			->insert('o_#__csrf', [
-				'username' => empty($userName) ? '-' : $userName,
+				'username' => $userName,
 				'date' => date('Y-m-d H:i:s'),
-				'ip' => \App\RequestUtil::getRemoteIP(),
-				'referer' => \App\Request::_getServer('HTTP_REFERER', '-'),
-				'url' => \App\RequestUtil::getBrowserInfo()->url,
-				'agent' => \App\Request::_getServer('HTTP_USER_AGENT', '-'),
+				'ip' => \App\TextParser::textTruncate(\App\RequestUtil::getRemoteIP(), 100),
```

Comment:
to function \App\TextParser::textTruncate set the third argument as false, everywhere in this function

Reply:
added